usaspending_api/download/models/award_financial.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from usaspending_api.awards.models import AbstractFinancialAccountsByAwards
+
+
+class AwardFinancialView(AbstractFinancialAccountsByAwards):
```

Comment:
One question is the names of the views.
I think i remember seeing a comment somewhere saying "one day, we'll finally get all these dabs tables/models named with the same convention" and i think it was referring to the `FinancialAccountsBy` convention `financial_accounts_by_`
And maybe the holdover to fix was file A.
So maybe think about changing the view model and view names to all start with `FinancialAccountsBy...` and `vw_financial_accounts_by_` resp.

And then for the views, keeping convention of the vw_ prefix. But differentiating the fact that these are download-specific views with a suffix
`vw_financial_accounts_by_awards_downloads` or `vw_financial_accounts_by_awards_dl`

(if there's a strong argument to lean back towards the broker naming convention for these tables -- i'm good to hear that too)

Reply:
Can get those changed over. Wanted the views to match the table they are based off of, but if we are planning to eventually change those tables then I am happy to make this change now for the new Views..